tests/integration-tests/tests/tests-test.js

```diff
@@ -55,6 +57,27 @@ describe('the tests api', function() {
             res.statusCode.should.eql(400);
             res.body.message.should.eql('processor with id: 123e4567-e89b-12d3-a456-426655440000 does not exist');
         });
+        it('Should return error when using functions not from processor', async () => {
+            const processor = {
+                name: 'some-user-processor: ' + uuid(),
+                description: 'This is a description',
+                javascript: 'module.exports.func = 5'
+            };
+            const processorResponse = await processorsRequestSender.createProcessor(processor, validHeaders);
+            processorResponse.statusCode.should.eql(201);
+            const processorId = processorResponse.body.id;
+            let requestBody = Object.assign({ processor_id: processorId }, testWithFunctions);
+            const createTestResponse = await testsRequestSender.createTest(requestBody, validHeaders);
+            createTestResponse.statusCode.should.eql(400);
+            createTestResponse.body.message.should.eql('Functions: beforeScenario, afterScenario, afterResponse, beforeRequest does not exist in the processor file');
```

Comment:
shouldn't this error be 
```suggestion
            createTestResponse.body.message.should.eql('Functions: beforeScenario, afterScenario, afterResponse, beforeRequest, abc does not exist in the processor file');
```

Reply:
you are right
we currently not supporting arrays also not in the ui
I will open a task to do it in the future

Reply:
so lets update the test_with_functions file